digester.go

```diff
@@ -72,6 +88,19 @@ type sqlDigester struct {
 	tokens tokenDeque
 }
 
+func (d *sqlDigester) doDigestNormalized(normalized string) (result string) {
+	hdr := *(*reflect.StringHeader)(unsafe.Pointer(&normalized))
+	b := *(*[]byte)(unsafe.Pointer(&reflect.SliceHeader{
+		Data: hdr.Data,
+		Len:  hdr.Len,
+		Cap:  hdr.Len,
+	}))
+	d.hasher.Write(b)
```

Comment:
Are you sure this is anything different from `d.hasher.Write([]byte(normalized))`? ([`io.WriteString`](https://golang.org/src/io/io.go#L289) also implements this just by a direct `[]byte` cast.)

Reply:
Here's an example https://play.golang.org/p/cYoqQ-VuibO, please run it locally.